skins/cat17/src/app/tests/components/BankData.spec.js

```diff
@@ -165,17 +173,170 @@ test( 'Given filled classic values on blur, classic validation is triggered and
 		} )
 	} );
 
-	const ibanInput = wrapper.find( '#account-id' );
-	const bicInput = wrapper.find( '#bank-id' );
+	const accountIdInput = wrapper.find( '#account-id' );
+	const bankIdInput = wrapper.find( '#bank-id' );
+
+	accountIdInput.setValue( '0648489890' );
+	accountIdInput.trigger( 'input' );
+	bankIdInput.setValue( '50010517' );
+	bankIdInput.trigger( 'input' );
+
+	accountIdInput.trigger( 'blur' );
+} );
+
+/* Setting of "valid" CSS class */
+
+test( 'Given empty fields, they are not marked as valid ', t => {
+	t.plan( 2 );
+
+	const wrapper = shallowMount( BankData, {
+		propsData: newTestProperties()
+	} );
+
+	const accountIdContainer = wrapper.find( '.field-account-id' );
+	const bankIdContainer = wrapper.find( '.field-bank-id' );
+
+	t.equal( accountIdContainer.classes().indexOf( 'valid' ), -1, 'empty account id field should not be valid' );
+	t.equal( bankIdContainer.classes().indexOf( 'valid' ), -1, 'empty bank id field should not be valid' );
+} );
+
+function fillField( wrapper, name, value ) {
+	const field = wrapper.find( name );
+	field.setValue( value );
+	field.trigger( 'input' );
+	field.trigger( 'blur' );
+}
+
+test( 'Given filled fields, they are marked as valid ', t => {
+	// Note: the class name "valid" is an unfortunate term, there for historical reasons.
```

Comment:
I like that you put more than one comment regarding that "problem" throughout the code.
This way it's more in our face and we won't wonder what we did there 6 months from now.